integration/common

```diff
@@ -80,6 +80,20 @@ launch_gethnet() {
   title "Geth is running."
 }
 
+launch_parity() {
+  printf -- "\033[34m%-50s\033[0m" "Checking for running ethereum node"
```

Comment:
Copy and paste error

Reply:
What should be the correct copy? Seems almost identical to line 68 which was the intention.